python/cuspatial/cuspatial/core/gis.py

```diff
@@ -21,53 +21,61 @@ def directed_hausdorff_distance(xs, ys, points_per_space):
     """Compute the directed Hausdorff distances between all pairs of
     spaces.
 
-    params
-    xs: x-coordinates
-    ys: y-coordinates
-    points_per_space: number of points in each space
-
     Parameters
     ----------
-    {params}
+    xs
+        column of x-coordinates
+    ys
+        column of y-coordinates
+    points_per_space
+        number of points in each space
 
-    Example
+    Returns
     -------
+    result : cudf.DataFrame
+        The pairwise directed distance matrix with one row and one
+        column per input space; the value at row i, column j represents the
```

Comment:
Did we verify this is correct (aka not vice versa)? I never did. I wrote it so the existing tests would pass, and I believe the existing docs state it this way, but I never confirmed the existing docs were correct. :/